src/semantic_digital_twin/world_description/world_state.py

```diff
@@ -180,8 +229,78 @@ def __deepcopy__(self, memo):
         """
         Create a deep copy of the WorldState.
         """
-        new_state = WorldState()
+        new_state = WorldState(_world=self._world)
         new_state.data = self.data.copy()
         new_state._names = self._names.copy()
         new_state._index = self._index.copy()
         return new_state
+
+    def add_degree_of_freedom_position(self, dof: DegreeOfFreedom):
```

Comment:
rename to `add_degree_of_freedom`.
i don't like the `_position` subfix. there is logic to set the initial position, but there has to be. it just doesn't make sense to allow a position outside the limits.
for the type of logic used to set the limits, the doc string is good enough.